internal/node/launcher.sh

```diff
@@ -257,24 +252,80 @@ if [[ "$PWD" == *"/bazel-out/"* ]]; then
   readonly rest=${PWD#*$bazel_out}
   readonly index=$(( ${#PWD} - ${#rest} - ${#bazel_out} ))
   if [[ ${index} < 0 ]]; then
-    echo "No 'bazel-out' folder found in path '${PWD}'!"
+    echo "No 'bazel-out' folder found in path '${PWD}'!" >&2
```

Comment:
nit: I saw a trick in the bazel sources that was
`echo >&2 other arguments here`
which seems neat as a way to say "echo to stderr"
but if we like that we should probably make a separate change